features/help_me_choose.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Help me choose
+
+  As a user, i want to use search filters, because they give a quick and easy way to find a car that suits me.
+
+  Background:
+    Given help me choose page is opened
+
+  @bodystyle
+  Scenario Outline: Browse by body style
+    When user choose <body_style> filter
+    Then user is redirected to a search page with selected <body_style> filter
+
+    Examples:
+      | body_style  |
+      | Hatchback   |
+      | Saloon      |
+      | Estate      |
+      | MPV         |
+      | SUV         |
+      | Convertible |
+      | Coupe       |
+
+  @lifestyle
+  Scenario Outline: Browse by lifestyle
+    When user choose <lifestyle> filter
+    Then user is redirected to a search page with selected <body_type> filter
+      And an Price range filter is seted as <price_range>
```

Comment:
Why additional trailing spaces on `And` steps?

Reply:
Corrected.